liquidator/liquidator.js

```diff
@@ -53,7 +77,11 @@ class Liquidator {
         lockedCollateral: response.events.LiquidationCreated.returnValues.lockedCollateral,
         liquidatedCollateral: response.events.LiquidationCreated.returnValues.liquidatedCollateral
       };
-      console.log(logResult);
+      Logger.info({
+        at: "liquidator",
+        message: "liquidation tx result",
+        liquidationResult: logResult
```

Comment:
Is it worth specifying the shape of this JSON in a separate file? In case we want to display liquidation results in a different file, we can make sure that the shape is consistent